### PR TITLE
Add high availability

### LINKED ISSUE #100: Doc High Availability example
For reference see: https://consensys.slack.com/archives/C0272B5P1CY/p1628776818024900

### LINKED ISSUE #109: Document High Availability (HA)
You can configure Tessera HA similar o Orion: https://docs.orion.consensys.net/en/stable/HowTo/High-Availability/

There's some content in the GoQuorum docs, but Besu users will bot be able to find this. See: https://docs.goquorum.consensys.net/en/latest/HowTo/Configure/HighAvailability/#tessera-ha-configuration-requirements

Perhaps also raise a separate Besu issue to link to the docs when you're done.

### PR DESCRIPTION
**Pull request by rolandtyler**
## Pull Request Description
Added High Availability page and example.

## Fixed Issue(s)

Fixes #100 
Fixes #109 

## Preview
https://pegasys-tessera--113.com.readthedocs.build/en/113/

### CODE REVIEW

**Review by joshuafernandes — APPROVED** [review]
LGTM, nice work @rolandtyler

File: docs/HowTo/Configure/High-availability.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+description: Tessera deployment for high availability
+---
+
+# High availability
+
+Tessera supports deploying more than one instance sharing the same database.
+
+By placing the instances behind a load balancer, downtime can be limited during maintenance operations.
+
+![Tessera-HA](../../images/tessera/Tessera-HA.png)
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:11]: Nitpick, but can you lessen the gray space at the top of the diagram and make "Balancer" and "Client" lowercase?

File: docs/HowTo/Configure/High-availability.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+## Servers
+
+Tessera exposes multiple interfaces for connectivity which can be configured in the `serverConfigs` in the
+[configuration file](../../Reference/SampleConfiguration.md).
+To enable High availability for the node, set the interfaces to use the load balancer's address for their `serverAddress`,
+and their own URL or IP for the `bindingAddress`, as shown in the following example.
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:18]:
```suggestion
To enable high availability for the node, set each interface to use the load balancer's address for its `serverAddress`,
and its own URL or IP for the `bindingAddress`, as shown in the following example.
```

File: docs/HowTo/Configure/High-availability.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+```bash
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:20]:
```suggestion
!!! example "Server configuration"

    ```bash
```

File: docs/HowTo/Configure/High-availability.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+---
+description: Tessera deployment for high availability
+---
+
+# High availability
+
+Tessera supports deploying more than one instance sharing the same database.
+
+By placing the instances behind a load balancer, downtime can be limited during maintenance operations.
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:9]:
```suggestion
By placing the instances behind a load balancer, downtime can be limited during maintenance operations, achieving high availability (HA).
```

File: docs/HowTo/Configure/High-availability.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+            }
+        }
+     }
+    ```
+
+The configuration defines two upstreams: `tessera_tp_9080` and `tessera_q2t_9101`.
+
+Both upstreams define health checks: `max_fails=3 fail_timeout=5s`
+
+The [health checks](https://docs.nginx.com/nginx/admin-guide/load-balancer/http-health-check/) help Nginx balance
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:119]:
```suggestion
The configuration defines two upstreams, `tessera_tp_9080` and `tessera_q2t_9101`, both of which define [health checks](https://docs.nginx.com/nginx/admin-guide/load-balancer/http-health-check/), `max_fails=3 fail_timeout=5s`.

The health checks help Nginx balance
```

File: docs/HowTo/Configure/High-availability.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+traffic among upstream servers.
+
+## Database
+
+The last piece to configure in High Availability is the [database](./Database.md).
```
Comment (alexandratran) [inline docs/HowTo/Configure/High-availability.md:124]:
```suggestion
The last piece to configure in high availability is the [database](./Database.md).
```

File: mkdocs.yml [hunk 7]
```diff
@@ -91,6 +91,7 @@ nav:
       - Cryptographic elliptic curves: HowTo/Configure/Cryptographic-elliptic-curves.md
       - Hyperledger Besu support: HowTo/Configure/Orion-Mode.md
       - Override configuration file: HowTo/Configure/Override-config.md
+      - High Availability: HowTo/Configure/High-availability.md
```
Comment (alexandratran) [inline mkdocs.yml:94]:
```suggestion
      - High availability: HowTo/Configure/High-availability.md
```

**Review by alexandratran — COMMENTED** [review]
Added some suggestions.

**Review by alexandratran — APPROVED** [review]
LGTM!

**Pull request merged**